pkgs/games/starsector/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, alsa-lib
+, fetchzip
+, libXxf86vm
+, makeWrapper
+, openjdk8
+, stdenv
+, xorg
+}:
+
+let
+  openjdk = openjdk8;
+in
```

Comment:
Please use an override instead.

Reply:
I thought it was fine. Like in https://github.com/NixOS/nixpkgs/blob/29cbec943d59b1ccfad0177e90b1d217d6571033/pkgs/applications/misc/obsidian/default.nix#L5

Do you mean an override in pkgs/top-level/all-packages.nix?

Reply:
> I thought it was fine. Like in 

well, kinda, it would be better if it would be done in all-packages.

> Do you mean an override in pkgs/top-level/all-packages.nix?

yes

Reply:
Like this?

https://github.com/NixOS/nixpkgs/pull/140064/files#diff-ab5748dc9567516fefba8344056b51ec1866adeace380f46e58a7af3d619ea22R9291-R9293